src/libraries/System.Drawing.Common/src/System/Drawing/Imaging/EncoderParameterPrivate.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices;
+using Gdip = System.Drawing.SafeNativeMethods.Gdip;
+
+namespace System.Drawing.Imaging
+{
+    [StructLayout(LayoutKind.Sequential)]
+    internal struct EncoderParameterPrivate
+    {
+        public Guid ParameterGuid;                    // GUID of the parameter
+        public readonly int NumberOfValues;           // Number of the parameter values
+        public readonly EncoderParameterValueType ParameterValueType;   // Value type, like ValueTypeLONG  etc.
```

Comment:
```suggestion
        public int NumberOfValues;                    // Number of the parameter values
        public EncoderParameterValueType ParameterValueType; // Value type, like ValueTypeLONG  etc.
```
`readonly` does not add much value here.